charts/lerian-notification/templates/secret.yaml

```
@@ -0,0 +1,23 @@
+*/}}
+{{- if not .Values.secretRef.name }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ include "lerian-notification.secretName" . }}
+  namespace: {{ include "lerian-notification.namespace" . }}
+  labels:
+    {{- include "lerian-notification.labels" (dict "context" . "name" (include "lerian-notification.fullname" .)) | nindent 4 }}
+type: Opaque
+data:
+  {{- range $key, $value := .Values.secrets }}
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)